examples/tfengine/generated/devops/cicd/configs/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
```

Comment:
set -ex

Reply:
I made the change in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/459 so I can merge it into this branch more easily